conf/distro/include/fslc-base.inc

```diff
@@ -43,3 +43,6 @@ PACKAGECONFIG:remove:pn-xserver-xorg:armv5 = "dri"
 # Log information on images and packages
 INHERIT += "buildhistory"
 BUILDHISTORY_COMMIT ?= "1"
+
+# imx-gpu-sdk requires older fmt
+PREFERRED_VERSION_fmt = "8.1.1"
```

Comment:
Really required?

Reply:
Yes. Otherwise there is a build break:

```
| /opt/work/upstream/fsl-xwayland/tmp/work/cortexa53-crypto-mx8mp-fsl-linux/imx-gpu-sdk/5.9.2-r0/git/DemoFramework/FslBase/include/FslBase/Log/Log3Fmt.hpp: In instantiation of 'void Fsl::Logger::WriteLine(Fsl::LogType, const char*, const Args& ...) [with Args = {unsigned int}]':
| /opt/work/upstream/fsl-xwayland/tmp/work/cortexa53-crypto-mx8mp-fsl-linux/imx-gpu-sdk/5.9.2-r0/git/DemoApps/GLES3/OpenCL101/source/OpenCL101.cpp:53:7:   required from here
| /opt/work/upstream/fsl-xwayland/tmp/work/cortexa53-crypto-mx8mp-fsl-linux/imx-gpu-sdk/5.9.2-r0/git/DemoFramework/FslBase/include/FslBase/Log/Log3Fmt.hpp:52:23: error: no matching function for call to 'format_to(fmt::v9::memory_buffer&, const char* const&, const unsigned int&)'
|    52 |         fmt::format_to(buf, pszFormat, args...);
|       |         ~~~~~~~~~~~~~~^~~~~~~~~~~~~~~~~~~~~~~~~
```

Reply:
It'd be better to apply a patch fixing it, seems not hard to fix. Could you take a look?

Reply:
It was not the only error. I just built again and got a different error.

Let me ask Rene if he can make a patch.

Reply:
Rene made 6.0.2 to fix this.